### DISCUSSION TITLE
Insert one json object into another?

### DISCUSSION THREAD

**Discussion by luz-arreola** [nlohmann/json#3930] [Q&A] [answered]
I would like to insert one json object into another.  Does Nlohmann have functionality built-in to handle this easily or do we have to manually iterate and merge?  For example, here are the two json objects:

```
{
	"a": 1,
	"b": {
		"product": "whatever",
		"items": [0, 1, 2]
	}
}

{
	"c": 3,
	"d": 
	[
		{"name": "andy"},
		{"age": 60},
		{"phone": "555-5555"}
	],
	"e": 5
}
```

Desired result:

```
{
	"a": 1,
	"b": {
		"product": "whatever",
		"items": [0, 1, 2]
	}
	"c": 3,
	"d": 
	[
		{"name": "andy"},
		{"age": 60},
		{"phone": "555-5555"}
	],
	"e": 5
}
```

Thank you!

**Comment by nlohmann** [comment, marked as answer]
See https://json.nlohmann.me/api/basic_json/update/

**Reply by luz-arreola** [reply]
Worked flawlessly!!  Thank you!!!
